src/objects.js

```diff
@@ -1,41 +1,50 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age
+  };
```

Comment:
if the name of the variable matches the name you want to give the key, you can just pass the variable instead of explicitly giving the key a name e.g.
```
const createPerson = (name, age) => {
  return {
    name,
    age
  };
```
this will create an object with 2 properties. The keys will have the names `name` and `age` and the values of these properties will be the value passed to the function